instrumentation/instrumentation.go

```diff
@@ -0,0 +1,20 @@
+// package instrumentation providies utilities for instrumenting Go code.
+package instrumentation
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"go.opencensus.io/trace"
+)
+
+// NewNopCensusExporter creates a NoOp exporter for the OpenCensus Trace package.
+// This exporter can be used for tests/local developement and does not require the user to provide
+// authentication to a remote tracing API.
+func NewNopCensusExporter() *exporter { return &exporter{} }
+
+type exporter struct{}
+
+func (e *exporter) ExportSpan(sd *trace.SpanData) {
+	fmt.Fprintln(ioutil.Discard, sd)
```

Comment:
Does opencensus expect that all of the span data is read?

Reply:
@zwass not sure. I used ioutil.Discard because all examples out there show writing to a writer, but I didn't want to pollute the logs.

Reply:
I think it can probably be removed. I had the instinct that `sd` was going to be an `io.Reader`, but it doesn't seem to be, so I think this really is a noop.